tests/test_interface.rb

```diff
@@ -1043,7 +1043,7 @@ def test_interface_ipv4_arp_timeout
 
   def test_interface_ipv4_forwarding
     intf = interfaces[0]
-    interface_ethernet_default(intf)
+    interface_ethernet_default(interfaces_id[0])
     i = Interface.new(intf)
```

Comment:
`interfaces[0]` resolves to `"ethernet1/1"` while `interfaces_id[0]` resolves to `"1/1"`.

The code for setting an ethernet interface to default requires the `interfaces_id` value to work:

```ruby
   def interface_ethernet_default(ethernet_id)
     config("default interface ethernet #{ethernet_id}")
   end
```

I prefer changing the `interface_ethernet_default` method to use `interfaces[0]` as the required value, but I went with the established pattern first.